airflow/config_templates/default_airflow.cfg

```diff
@@ -422,6 +422,13 @@ backend =
 # ``{{"connections_prefix": "/airflow/connections", "profile_name": "default"}}``
 backend_kwargs =
 
+# Advanced secrets backend configuration, allow to user configure more than one secret backend,
```

Comment:
same comment as elsewhere; better have just one way then "a way" and "an advanced way"

Reply:
Agree.